lib/css/modules/_floater.scss

```diff
@@ -15,4 +16,8 @@
 	display: flex;
 	opacity: .7;
 	&:hover { opacity: 1; }
+	
```

Comment:
```
lib/css/modules/_floater.scss
  18:25  error  No trailing whitespace allowed          no-trailing-whitespace
  21:16  error  Don't include leading zeros on numbers  leading-zero
```

Hmm the `leading-zero` rule is kinda silly, you could turn it off (`.sass-lint.yml`).